packages/beacon-node/src/metrics/metrics/lodestar.ts

```diff
@@ -821,7 +821,16 @@ export function createLodestarMetrics(
         name: "validator_monitor_validators",
         help: "Count of validators that are specifically monitored by this beacon node",
       }),
-
+      validatorStatuses: register.gauge<"status">({
+        name: "validator_monitor_validators_status",
+        help: "Statuses of the various validators monitored by this beacon node",
+        labelNames: ["status"],
+      }),
+      validatorBalances: register.gauge<"index">({
```

Comment:
No difference, we can use the existing one.

Reply:
well since both of those use `index` label they are quite expensive, so keeping both does not make sense IMO and we should be careful with removing/renaming metrics. Would generally vote for keeping `prevEpochOnChainBalance` in that case